app/controllers/smart_answers_controller.rb

```diff
@@ -95,13 +97,23 @@ def set_header_footer_only
   end
 
   def set_expiry(duration = 30.minutes)
-    # if the artefact returned from the Content API is blank, or if
-    # the request to the Content API fails, set a very short cache so
-    # we don't cache an incomplete page for a while
-    duration = 5.seconds if @presenter.present? && @presenter.artefact.blank?
-
     if Rails.configuration.set_http_cache_control_expiry_time
       expires_in(duration, public: true)
     end
   end
+
+  def setup_navigation_helpers_and_content_item
+    @content_item = Services.content_store.content_item!("/" + params[:id]).to_hash
+
+    # Remove the organisations from the content item - this will prevent the
+    # govuk:analytics:organisations meta tag from being generated until there is
```

Comment:
Why don't we want this meta tag set? Reading this without any prior context, this comment is a little mysterious. Could do with a little more detail.